Jenkinsfile

```diff
@@ -0,0 +1,79 @@
+node("vdvs-slave-two") {
+
+    def app
+   
+    def ERROR = 'false'
+   
+    dir('docker-volume-vsphere') {
```

Comment:
let's get started using the new name and avoid referring `docker-volume-vsphere`